broker/staticconfiguration/impl.go

```diff
@@ -0,0 +1,46 @@
+package staticconfiguration
+
+import (
+	"errors"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+)
+
+const defaultServicePort = 443
+const defaultStatusPort = 6930
+const defaultAccountConfigurationUrl = "file:///etc/cloud-gate/accounts.yml"
+const defaultAccountConfigurationCheckInterval = "300s"
+
+func LoadVerifyConfigFile(configFilename string) (*StaticConfiguration, error) {
+	var config StaticConfiguration
+	if _, err := os.Stat(configFilename); os.IsNotExist(err) {
+		err = errors.New("mising config file failure")
+		return nil, err
+	}
+	source, err := ioutil.ReadFile(configFilename)
+	if err != nil {
+		err = errors.New("cannot read config file")
```

Comment:
Doing this masks the underlying error. The user can't tell if the file is missing, there's a permissions problem, lack of file descriptors, whatever.